spinalcordtoolbox/deepseg/models.py

```diff
@@ -95,23 +95,6 @@
         "contrasts": ["mp2rage"],
         "default": False,
     },
-    "model_seg_ms_lesion_mp2rage": {
-        "url": {
-            "seed1": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210/model_seg_lesion_mp2rage_r20230210_dil32_seed01.zip"],
-            "seed2": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210/model_seg_lesion_mp2rage_r20230210_dil32_seed02.zip"],
-            "seed3": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210/model_seg_lesion_mp2rage_r20230210_dil32_seed03.zip"],
-            "seed4": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210/model_seg_lesion_mp2rage_r20230210_dil32_seed04.zip"],
-            "seed5": ["https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20230210/model_seg_lesion_mp2rage_r20230210_dil32_seed05.zip"],
-        },
```

Comment:
I was curious how the "check if model is up to date" code would handle the transition from a multi-seed ensemble task to a single-seed task. But, it looks like it handles things just fine:

```console
(venv_sct) nlm/update-_seg_ms_lesion_mp2rage-model|joshua@monarch:~/repos/spinalcordtoolbox/data/sct_testing_data/t2$ sct_deepseg -task seg_ms_lesion_mp2rage -i t2.nii.gz

--
Spinal Cord Toolbox (git-nlm/update-_seg_ms_lesion_mp2rage-model-a82c72c5990f604d69ecee1667692d9aaeccf11e)

sct_deepseg -task seg_ms_lesion_mp2rage -i t2.nii.gz
--

Mismatch between 'source.json' URL format and SCT source code URLs
Model model_seg_ms_lesion_mp2rage is out of date. Re-installing it now...

INSTALLING MODEL: model_seg_ms_lesion_mp2rage
Removing existing destination folder '/home/joshua/repos/spinalcordtoolbox/data/deepseg_models/model_seg_ms_lesion_mp2rage'
Trying URL: https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20240610/nnUNetTrainer_seg_ms_lesion_mp2rage__nnUNetPlans__3d_fullres.zip
```

Great job to @mguaypaq for writing robust code that takes into account a potential mismatch in URL formats:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/a82c72c5990f604d69ecee1667692d9aaeccf11e/spinalcordtoolbox/deepseg/models.py#L462-L464